_docs/v2.1/setup/install.md

```diff
@@ -51,14 +51,14 @@ steps below:
       git clone https://github.com/GoogleCloudPlatform/forseti-security.git
       ```
 
-  1. Check out the specific version of Forseti you want to install by using a tag like `v2.0.0.`:
+  1. Check out the specific version of Forseti you want to install by using a tag like `v2.1.0.`:
```

Comment:
Hmmm, this was meant to serve as a generic example.  But I can see how there can be a problem for copy/paste.  However, this change would force us to update the version here on every release.  So how about this, like below?
```
Check out the specific version of Forseti you want to install by using a tag like `v2.1.0, v2.2.0, v2.3.0, etc.`.  You can visit the release page (add link) to see which versions are available to install:

git checkout tags/v<X.Y.Z>
```

Reply:
That makes sense to me. I was actually hoping to use another `sed` command in `create_new_version_from_latest.sh` to change this to the latest version automatically. I couldn't think of a way that didn't end up with a nasty `%VERSION%` macro or something like that.

Reply:
Cool, will you change the PR here to make it more generic?

Reply:
Will do. Try to get to it later today.